commitlint.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    extends: ['@commitlint/config-conventional'],
```

Comment:
Can we use this?
https://github.com/PSMRI/MMU-API/pull/74/files#diff-2b1ae4316086c0ebb213227790133a792300e1cef03b0afc067b23d3399ea5ca